### PR TITLE
Additions to EHR section

### PR DESCRIPTION
**Pull request by enricoferrero**
I'd like to make some additions to the EHR section. Specifically I cited #82 and #152 (which were already referenced elsewhere) and added #627. I also emphasised the visualisation aspect of #25 which I thought was very cool.

The paragraph should now work as a nice overview of predicting disease trajectories from EHR data using different architectures (starts with DA, then RNN and finishes with CNN). If those papers were left out of this section intentionally then feel free to ignore this pull request.

Requesting feedback/review from @cgreene and @brettbj.

Cheers,
Enrico

### CODE REVIEW

**Comment by agitter** [comment]
Thanks @enricoferrero.  I've been slow reviewing lately, but one of us will check this out.  I haven't looked yet, but does this affect any of the sections updated in #577?

Also, please make `arxiv` all lower case in the new references:
```
@arXiv:1511.05942 → source "arXiv" is not valid
@arXiv:1602.00357 → source "arXiv" is not valid
```

**Comment by enricoferrero** [comment]
Thanks @agitter - capitalisation fixed. And no, it should not conflict with #577.

File: sections/03_categorize.md [hunk 1]
```diff
@@ -294,22 +294,34 @@ processes to distinguish gout from leukemia from uric acid sequences. Later work
...
-tests, text-based patient records, image analysis, genomic data) and better
-handling of streaming temporal data with many features, will need to be overcome
-before we can fully assess the potential of deep learning for this application
-area.
+improved by up to 15% when compared to other methods. Choi et
+al. [@arxiv:1511.05942] attempted to model the longitudinal structure of EHRs
+with a RNN to predict future diagnosis and medication prescriptions on a cohort
+of 260,000 patients followed for 8 years (Doctor AI). Pham et al.
+[@arxiv:1602.00357] built upon this concept by utilising a RNN with a long 
+short-term memory (LSTM) architecture enabling explicit modelling of patient
```
Comment (agapow) [inline sections/03_categorize.md]: Don't we have LSTM defined somewhere earlier?

File: sections/03_categorize.md [hunk 2]
```diff
@@ -294,22 +294,34 @@ processes to distinguish gout from leukemia from uric acid sequences. Later work
...
+trajectories through the use of memory cells. The method, DeepCare, performed
+better than shallow models or plain RNN when tested on two independent cohorts
+for its ability to predict disease progression, intervention recommendation and
+future risk prediction. Nguyen et al. [@doi:10.1109/JBHI.2016.2633963] took a
+different approach and used word embeddings from EHRs to train a CNN that could
+detect and pool local clinical motifs to predict unplanned readmission after 6
```
Comment (agapow) [inline sections/03_categorize.md]: "six" not "6"?

File: sections/03_categorize.md (hunk 1 above)
Comment (enricoferrero) [reply sections/03_categorize.md]: Just checked and this is the first time it appears in the the version I was working on at the time.

File: sections/03_categorize.md (hunk 2 above)
Comment (enricoferrero) [reply sections/03_categorize.md]: Fixed

**Comment by enricoferrero** [comment]
Thanks @agapow. I pulled master to resolve the conflicts, it looks like it can be merged now!

File: sections/03_categorize.md (hunk 1 above)
Comment (enricoferrero) [reply sections/03_categorize.md]: Still is after merging.

**Comment by cgreene** [comment]
I had to look to see if we talk at all about embeddings elsewhere. We mention word2vec before this but we actually don't have much in the paper on embeddings or latent spaces (😨 the term appears only one time!) but that's probably an issue for another PR.

Tagging @gwaygenomics in case we want to think about discussing latent spaces in the discussion section as an area of future interest via a paragraph, but I think these changes are ready to merge so I'm going to go ahead with them.

**Pull request merged**